### PR TITLE
`\dmsg-discovery\all_servers` endpoint

### LINKED ISSUE #155: `/all_servers` endpoint
**Feature description**
It should be good if we have another endpoints like `all_server` that work same as `/available_servers` to show all dmsg-servers, not availables. It's useful for dmsghttp-config.json file.
Also we will need to use this new endpoint in other services instead of `/available_servers` as we want to serve the services on all dmsg-servers and not just available ones.

### PR DESCRIPTION
**Pull request by mrpalide**
Fixes #155 

 Changes:	
- add new endpoint `/dmsg-discovery/all_servers` to get all dmsg servers list
- make new AllServers api endpoint for directClient and other servicess usage

How to test this PR:
_

### CODE REVIEW

File: disc/testing.go [hunk 1]
```diff
@@ -129,7 +129,7 @@ func (m *mockClient) PutEntry(ctx context.Context, sk cipher.SecKey, e *Entry) e
 	}
 }
 
-// AvailableServers returns all the servers that the APIClient mock has
+// AvailableServers returns all available the servers that the APIClient mock has
```
Comment (ersonp) [inline disc/testing.go]: Change `available the servers` to `available servers`

File: cmd/dmsg-discovery/README.md [hunk 2]
```diff
@@ -169,3 +169,39 @@ Possible Status Codes:
 - Forbidden (403) - When access is forbidden.
 
 - Internal Server Error (500) - Something unexpected happened.
+
+### GET Available Servers
```
Comment (ersonp) [inline cmd/dmsg-discovery/README.md]: This should be `GET All Servers`

File: cmd/dmsg-discovery/internal/api/api.go [hunk 3]
```diff
@@ -294,6 +295,30 @@ func (a *API) getAvailableServers() http.HandlerFunc {
 	}
 }
 
+// getAllServers returns all servers entries as am array of json codified entry objects
```
Comment (ersonp) [inline cmd/dmsg-discovery/internal/api/api.go]: Change `entries as am array` to`entries as an array`

File: disc/testing.go (hunk 1 above)
Comment (mrpalide) [reply disc/testing.go]: Done.

File: cmd/dmsg-discovery/internal/api/api.go (hunk 3 above)
Comment (mrpalide) [reply cmd/dmsg-discovery/internal/api/api.go]: Done.

File: cmd/dmsg-discovery/README.md (hunk 2 above)
Comment (mrpalide) [reply cmd/dmsg-discovery/README.md]: Done.

**Review by ersonp — APPROVED** [review]
Code looks good and everything is working well in the integration env.

**Pull request merged**